mage/changelog.md

```diff
@@ -4,6 +4,87 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## v1.4 - Dec 13, 2021
+
+### Major Features and Improvements
+
+- Implement Link prediction with DGL.
```

Comment:
Suggest we either replace DGL with the full name or convert it to a link as follows: [DGL](https://www.dgl.ai/).